subsys/bluetooth/host/ssp.c

```diff
@@ -779,7 +779,7 @@ void bt_hci_auth_complete(struct net_buf *buf)
 
 	LOG_DBG("status 0x%02x, handle %u", evt->status, handle);
 
-	conn = bt_conn_lookup_handle(handle);
+	conn = bt_conn_lookup_handle(handle, BT_CONN_TYPE_ALL);
```

Comment:
Everything in `ssp.c` is specific to BR/EDR, so this should be `BT_CONN_TYPE_BR`